src/ripple/protocol/impl/TER.cpp

```diff
@@ -126,6 +133,7 @@ transResults()
         MAKE_ERROR(telCAN_NOT_QUEUE_FULL,     "Can not queue at this time: queue is full."),
 
         MAKE_ERROR(temMALFORMED,                 "Malformed transaction."),
+        MAKE_ERROR(temAMM_BAD_TOKENS,            "Malformed: Invalid LPTokens."),
```

Comment:
Suggestion: rename this to `temBAD_AMM_TOKENS` to fit the pattern of the other BAD ter codes.